app/views/users/show.html.erb

```diff
@@ -15,10 +15,25 @@
 
 <div class="activity-feed">
   <h3>Mon activité :</h3>
-  <% @tracks.each do |track| %>
+  <% @activities.each do |activity| %>
```

Comment:
j'ai cette erreur quand je teste ActiveRecord::AssociationNotFoundError: Association named 'user_track' was not found on Post; perhaps you misspelled it?